package.json

```diff
@@ -35,7 +38,7 @@
     "serialize-javascript": "^1.4.0",
     "schema-utils": "^0.3.0",
     "source-map": "^0.6.1",
-    "uglify-es": "3.2.2",
+    "uglify-es": "^3.3.3",
```

Comment:
Any particular reason 3.3.3 was chosen?
```
     '3.2.2': '2017-12-10T18:36:46.421Z',
     '3.3.0': '2017-12-24T16:05:47.966Z',
     '3.3.1': '2017-12-25T13:11:01.187Z',
     '3.3.2': '2017-12-25T21:24:34.232Z',
     '3.3.3': '2017-12-28T21:32:12.606Z',
     '3.3.4': '2017-12-31T04:57:50.883Z' },
```
https://github.com/mishoo/UglifyJS2/commits/harmony

Reply:
https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/202#issuecomment-355142620 is the only 'reason',  I will update to `v3.3.4` (@latest) :)

Reply:
done